xyzprinting_antibact_pla.xml.fdm_material

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>XYZprinting</brand>
+      <material>Anti-Bact</material>
```

Comment:
What type should this be? Anti-bat isn't a plastic that I know ;)

Reply:
It is PLA, Anti-Bacteria PLA
Thank you :)